modules/Router.js

```diff
@@ -15,5 +15,64 @@
    * @class  Router
    * @author kent.willis@vecna.com
    */
-  return Backbone.Router.extend({});
+
+  var Router = Backbone.Router.extend({
+
+    constructor: function(){
+      Backbone.Router.apply(this, arguments);
+      this.__applyTimingEventListeners(['pre-route', 'post-route']);
```

Comment:
@cmchin, I can't think of one. Although, I could see you passing in pairs of strings to which it can handle Events-specific logic across events (passing information from the start to the stop if needed).

I like being declarative about which events we are binding to timing events, at least for now.